lib/sqlalchemy/sql/schema.py

```diff
@@ -126,6 +126,10 @@
     "FetchedValue", str, TextClause, ColumnElement[Any]
 ]
 
+_ServerOnUpdateArgument = Union[
```

Comment:
Maybe we can re-use the `_ServerDefaultArgument` type (could be renamed)

@zzzeek do you prefer having two aliases here or just one?

Reply:
Only hard thing about that is finding a good name for it

Reply:
that works too

Reply:
Updated!